library/bitmap-library/src/main/java/uk/gov/gchq/gaffer/bitmap/serialisation/StringKeyedMapOfBitmapsSerialiser.java

```
@@ -0,0 +1,86 @@
+package uk.gov.gchq.gaffer.bitmap.serialisation;
+
+import org.roaringbitmap.RoaringBitmap;
+import uk.gov.gchq.gaffer.bitmap.types.MapOfBitmaps;
+import uk.gov.gchq.gaffer.exception.SerialisationException;
+import uk.gov.gchq.gaffer.serialisation.ToBytesSerialiser;
+import uk.gov.gchq.gaffer.serialisation.implementation.StringSerialiser;
+import uk.gov.gchq.gaffer.serialisation.implementation.raw.CompactRawSerialisationUtils;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Map;
+
+public class StringKeyedMapOfBitmapsSerialiser implements ToBytesSerialiser<MapOfBitmaps> {
+
+    private static final RoaringBitmapSerialiser BITMAP_SERIALISER = new RoaringBitmapSerialiser();
+    private static final StringSerialiser STRING_SERIALISER = new StringSerialiser();
+
+    @Override
+    public boolean canHandle(Class clazz) {
+        return MapOfBitmaps.class.equals(clazz);
+    }
+
+    @Override
+    public byte[] serialise(MapOfBitmaps object) throws SerialisationException {
+        ByteArrayOutputStream byteOut = new ByteArrayOutputStream();
+        try {
+            for(Map.Entry<Object, RoaringBitmap> entry : object.entrySet()) {
+                byte[] k = STRING_SERIALISER.serialise((String) entry.getKey());
```

Should be final - can we throw an exception if the key is not a string?